src/read/pe/resource.rs

```diff
@@ -166,6 +166,21 @@ impl ResourceName {
             .read_slice::<U16Bytes<LE>>(&mut offset, len.get(LE).into())
             .read_error("Invalid resource name length")
     }
+
+    /// Returns the string buffer as raw bytes.
+    pub fn raw_data<'data>(&self, directory: ResourceDirectory<'data>) -> Result<&'data [u8]> {
```

Comment:
Instead of duplicating logic, can we implement in terms of `data`. Something like `self.data(directory).map(pod::bytes_of_slice)`.

Reply:
Ah yes! I didn't know about bytes_of_slice

Reply:
Since bytes_of_slice is public, this oneliner can be done on the user (my) side, so this helper is not that very useful anymore. I'll let you decide if it makes sense to keep it or not